src/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
Can these be MDX? 

Also, the state of the two switches is linked, when viewing in 'docs' mode